akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala

```diff
@@ -373,13 +373,13 @@ object Source {
    * if there is no demand from downstream. When `bufferSize` is 0 the `overflowStrategy` does
    * not matter.
    *
-   * The stream can be completed successfully by sending the actor reference an [[akka.actor.Status.Success]]
-   * message in which case already buffered elements will be signaled before signaling completion,
-   * or by sending a [[akka.actor.PoisonPill]] in which case completion will be signaled immediately.
+   * The stream can be completed successfully by sending the actor reference a [[akka.actor.Status.Success]]
```

Comment:
Why? Doesn't that usually appear as "sending the actor reference a Success" (without qualification)?